beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -470,7 +470,7 @@ func UnslashedAttestingIndices(ctx context.Context, state state.ReadOnlyBeaconSt
 	seen := make(map[uint64]bool)
 
 	for _, att := range atts {
-		committee, err := helpers.BeaconCommitteeFromState(ctx, state, att.Data.Slot, att.Data.CommitteeIndex)
+		committee, err := helpers.BeaconCommitteeFromState(ctx, state, att.GetData().Slot, att.GetData().CommitteeIndex)
```

Comment:
Unrelated to the PR, but for the getter, it would be cleaner to shortcut the data part and just use `att.CommitteeIndex()`.

Reply:
Nice suggestion! I will open another PR with several of these